src/lib/chat/sendbird-client.ts

```diff
@@ -101,6 +101,14 @@ export class SendbirdClient implements IChatClient {
     return results.map((u) => ({ id: u.id, display: u.name, profileImage: u.profileImage }));
   }
 
+  async deleteMessageByRoomId(channelId: string, messageId: number): Promise<any> {
+    try {
+      await del<any>(`/chatChannels/${channelId}/message`).send({ message: { id: messageId } });
+    } catch (error: any) {
+      console.log('Error occurred while deleting message', error?.response ?? error);
+    }
```

Comment:
It's not a good idea to just swallow the error and do nothing. We should let this throw so that other code can decide what to do.

Reply:
Updated